_projects/autosegct.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: project_page
+title: Radiotherapy Auto Segmentation
+status: Evaluating
+image: /assets/img/projects/autosegct_logo.png
+summary: Single application for all in-house auto-segmentation contours
+csc-lead: <a href="/team_member/Anil.html">Anil Mistry</a> <a href="/team_member/Tom.html">Tom Roberts</a>
```

Comment:
```suggestion
csc-lead: <a href="/team_member/Anil.html">Anil</a> <a href="/team_member/Tom.html">Tom</a>
```

Suggested edit in keeping with [current standard](https://github.com/GSTT-CSC/gstt-csc.github.io/pull/353#issuecomment-1983713501).